### ISSUE TITLE
[Feature] Let actions load external script as `data`

### ISSUE THREAD

**Issue by zubairy** [windingwind/zotero-actions-tags#402]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

- OS: Windows 11 23H2
- Zotero Version: 7.0.7 (64-bit)
- Plugin Version: 2.0.3

### Describe the feature request

**Is your feature request related to a problem? Please describe.**
> A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

In #176, we can export Actions code to synchronize configurations between different computers. However, this still requires manual operation. Is it possible to achieve automatic synchronization, for example: 1) I store the written JavaScript code as an item attachment in Zotero; 2) Create an Action and read the content of this attachment as `data` to execute.

**Why do you need this feature?**
> A clear and concise description of why you need this feature.

In this way, I only need to update the JS code stored in Zotero, and use Zotero's synchronization function to seamlessly execute Actions in different terminals (Of course, the differences in compatibility with multiple terminal configurations must be taken into account in the JS code script). This is especially useful when I just start to create and debug a code, because at this time I often need to frequently modify and test code.

### Describe the solution you'd like

**The solution you'd like**
> A clear and concise description of what you want to happen.

As stated above, let Actions read external **synced** script as `data`, similar to the operation `Trigger Other Actions`, but `Exec External Script`.

**Alternatives you've considered**
> A clear and concise description of any alternative solutions or features you've considered.

In short, what I want is to achieve automatic synchronization of multi-terminal configurations of the Actions. Haven't alternative solutions yet.

**Comment by windingwind** [comment]
You can read and execute external JS files in action.

**Comment by zubairy** [comment]
It would be appreciated if you could leave a snippet, or some instructions, showing how to do it.

**Comment by zubairy** [comment]
Searched and know how, cheers.

> You can read and execute external JS files in action.
